File: ios/BugReport/ReportType.swift
import Foundation

public enum BugReportType: String, CaseIterable {
    case bug = "Bug"
    case suggestion = "Suggestion"
    case question = "Question"
    case crash = "Crash"
    
    public var icon: String {
        switch self {
        case .bug:
            return "🐞"
        case .suggestion:
            return "💡"
        case .crash:
            return "🚨"
        case .question:
            return "❓"
        }
    }
    
    public var displayText: String {
        return "\(icon) \(rawValue)"
    }
}

File: android/src/main/java/com/margelo/nitro/toadly/ReportType.kt
package com.margelo.nitro.toadly

enum class ReportType(val displayName: String, val icon: String) {
    BUG("Bug", "🐞"),
    SUGGESTION("Suggestion", "💡"),
    QUESTION("Question", "❓"),
    CRASH("Crash", "🚨");

    val displayText: String
        get() = "$icon $displayName"

    companion object {
        fun getDefault(): ReportType = BUG
    }
}
